main.tf

```diff
@@ -197,7 +201,7 @@ JSON
       format(
         "arn:${data.aws_partition.current.partition}:s3:::%s/%s/AWSLogs/%s/*",
```

Comment:
Wait, @cblkwell ...is it this line that's throwing things off? You changed the format string template in the line above but not here. This is where you'd get `//`

Reply:
Ugh, yep, I bet you're right.